client/signup/style.scss

```diff
@@ -72,7 +72,7 @@ body.is-section-signup .layout.gravatar {
 		position: relative;
 	}
 
-	.suggestions__wrapper {
```

Comment:
Will you add these other usages to the testing instructions. It's critical that we don't cause e.g. current signup to regress 😉

Reply:
Absolutely. I should have updated the description altogether (and run more tests). 😃